src/FamilyHubs.Notification.Api/Endpoints/MinimalNotifyEndPoints.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FamilyHubs.Notification.Core.Commands.CreateNotification;
+using FamilyHubs.Notification.Core.Queries.GetSentNotifications;
+using FamilyHubs.Notification.Api.Contracts;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace FamilyHubs.Notification.Api.Endpoints;
+
+public class MinimalNotifyEndPoints
+{
+    public void RegisterMinimalNotifyEndPoints(WebApplication app)
+    {
+        app.MapPost("api/notify", async ([FromBody] MessageDto request, CancellationToken cancellationToken, ISender _mediator) =>
```

Comment:
We should probably protect sending notifification with [Authorize] too.

Reply:
Added Authorize attribute